pyaedt/modules/PostProcessor.py

```diff
@@ -877,12 +877,12 @@ def copy_report_data(self, PlotName):
         return True
 
     @pyaedt_function_handler()
-    def delete_report(self, plot_name=None):
+    def delete_report(self, PlotName=None):
```

Comment:
I would suggest to keep the parameter name to `plot_name` to follow snake_case naming convention.
```suggestion
    def delete_report(self, plot_name=None):
```